girder-dandi-archive/tests/rest/test_stats.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+
+from girder.models.folder import Folder
+from girder.models.upload import Upload
+
+from pytest_girder.assertions import assertStatusOk
+
+pytestmark = pytest.mark.plugin("dandi_archive")
+path = "/dandi/stats"
+
+
+def test_stats_zero(server):
+    resp = server.request(path=path, method="GET")
+    assertStatusOk(resp)
+    assert {
+        "draft_count": 0,
+        "published_count": 0,
+        "user_count": 0,
+        "species_count": 0,
+        "subject_count": 0,
+        "cell_count": 0,
+        "size": 0,
+    } == resp.json
+
+
+def test_stats_user(server, user):
+    resp = server.request(path=path, method="GET")
+    assertStatusOk(resp)
+    assert {
+        "draft_count": 0,
+        "published_count": 0,
+        "user_count": 2,  # one admin user, one normal user
+        "species_count": 0,
+        "subject_count": 0,
+        "cell_count": 0,
+        "size": 0,
+    } == resp.json
+
+
+def test_stats_draft(server, user, dandiset_1):
```

Comment:
Does this test need a `user`?

Reply:
No, with caveats. I added it because dandiset_1 requires a user, with means `user_count` must be 2.
I was torn between including it to clearly signal why `user_count=2`, or excluding because it's unnecessary and leaving that 2 as a magic number. If there are any feelings one way or the other I'm happy to change it.

Reply:
This would be great to have a brief code comment to explain.